framework/src/org/checkerframework/common/value/qual/ArrayLen.java

```diff
@@ -7,9 +7,9 @@
 import org.checkerframework.framework.qual.SubtypeOf;
```

Comment:
Because `ArrayLen` now applies to Strings, we should probably rename it. Perhaps to `@Len`? Or `@Length`?